### PR TITLE
Add ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by dmikurube**
Since we are recently observing more unreadable issues and/or feature requests to be for plugins, let me set the `ISSUE_TEMPLATE` for filing new issues.

@sakama @kamatama41 Can you have a look? (Cc: @hiroyuki-sato)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Issue Type: New Feature Request
+--------------------------------
+
+* Write your concrete practical scenarios with the new feature you want.
+* Write your expectation how your suggestion helps others as well.
+* Do not file a request like "support a new input/output/filter" here.
+  * That's not for here.
+  * Embulk is plugin-based. Such inputs/outputs/filters are to be implemented as plugins, not in the core.
+  * It is recommended for you to search for existing plugins at first.
```
Comment (hiroyuki-sato) [inline .github/ISSUE_TEMPLATE.md]: What do you think to add this URL here http://www.embulk.org/plugins/ ?

Comment (dmikurube) [reply .github/ISSUE_TEMPLATE.md]: Thanks. Sounds like a good idea. :)

**Comment by hiroyuki-sato** [comment]
LGTM 👍

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Issue Type: Bug Report
+-----------------------
+
+* Write the following environmental information.
+  * OS version
+  * Java version
+  * Embulk version
+  * Your Embulk configuration (YAML)
+  * Plugin versions
+* Write all what you did, e.g. your commends executed
```
Comment (sakama) [inline .github/ISSUE_TEMPLATE.md]: typo: commands

Comment (dmikurube) [reply .github/ISSUE_TEMPLATE.md]: Good catch! Thanks.

**Comment by dmikurube** [comment]
@kamatama41 I'll be merging this soon. Please ping me if you see any concerns.

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**